m2c/arch_mips.py

```diff
@@ -957,6 +961,8 @@ def make_memory_access(arg: Argument) -> List[Location]:
                 loc = StackLocation.from_offset(arg.lhs)
                 if loc is None:
                     return []
+                elif size == 16:
+                    return [loc, replace(loc, offset=loc.offset + 8)]
                 elif size == 8:
                     return [loc, replace(loc, offset=loc.offset + 4)]
```

Comment:
I want to bring attention here. I'm not sure if this is correct, I fiddled with it until m2c stopped complaining about gaps on the stack

Reply:
My guess is that `[loc, replace(loc, offset=loc.offset + 4), replace(loc, offset=loc.offset + 8), replace(loc, offset=loc.offset + 12)]` is the right thing to return here